### PR TITLE
Show example of checkout page customization

### PR DESCRIPTION
**Pull request by NicolasDorier**
Need their logo and @astupidmoose logo/css as well! :)

### CODE REVIEW

**Comment by astupidmoose** [comment]
https://coincards.com/btcpay.css

**Comment by NicolasDorier** [comment]
@astupidmoose and logo?

**Comment by NicolasDorier** [comment]
Can you put a screenshot here?

**Comment by astupidmoose** [comment]
Logo: https://assets.coincards.com/wp-content/uploads/2019/04/08121831/coincards_logo_global_medium.png

Like this? 
image

File: Theme.md [hunk 1]
```diff
@@ -30,3 +30,61 @@ Now, go into the settings of your store and paste the link to customize the boot
 And enjoy a new theme!
 
 ![CustomTheme](img/CustomTheme.png)
+
+## Checkout page theme
+
+It is possible to entirely customize the checkout experience of your customer with your brand.
+
+Go to you Store Settings, Checkout Experience, then follow those examples:
```
Comment (britttttk) [inline Theme.md]: Go to your Store Settings, Checkout Experience, Link to a custom CSS stylesheet. Create your custom theme using these examples below:

Maybe we can add something like:

Server admins can upload their .css file to [File Storage](https://github.com/btcpayserver/btcpayserver-doc/blob/master/FAQ/FAQ-ServerSettings.md#how-to-upload-files-to-btcpay) to create a CSS stylesheet link.

**Comment by pavlenex** [comment]
Blockstream also has a sick looking checkout page in their store.
Screen Shot 2019-06-07 at 08 30 52

**Comment by pavlenex** [comment]
We do have something in FAQ already  about modifing the checkout page, don't mind

https://docs.btcpayserver.org/faq-and-common-issues/faq-serversettings#how-to-modify-the-checkout-page

**Comment by NicolasDorier** [comment]
@pavlenex can you get the logo of blockstream? Right click inspect

**Comment by pavlenex** [comment]
@NicolasDorier you mean their CSS?

Sure:

```
html {
  background-color: #111316 !important;
}

.modal.page {
  background-color: #111316;
}

.top-header {
  background-color: #15181c;
}

.top-header .timer-row__progress-bar {
  background: #00c3ff;
}

.top-header .timer-row {
  background: #1187c1;
}

.payment-tabs__slider {
  background: #00c3ff;
}

.action-button {
  color: #fff !important;
  background-image: linear-gradient(to right, #00c3ff, #1187c1);
}

.action-button:active {
  background-image: linear-gradient(to right, #00c3ff, #1187c1);
}

.invoice {
  background-color: #15181c;
}

.expired__body {
  color: #fff;
}

.currency-selection {
  background-color: #15181c;
  border-bottom: 1px solid #565d6e;
  color: #fff;
  padding-top: 15px;
  padding-bottom: 15px;
  height: auto;
}

.line-items {
  background-color: #111316;
  color: #565d6e !important;
  border-top: 1px solid #565d6e !important;
}

.buyerTotalLine {
  border-top: 1px solid #202224;
  background-color: #15181c;
  color: #fff;
}

.payment-tabs {
  background-color: #15181c;
  color: #fff;
  border-top: 1px solid #565d6e;
  border-bottom: 1px solid #565d6e;
}

.payment-tabs__tab .active {
  color: #00c3ff;
}

.payment-tabs__tab.active {
  color: #00c3ff;
}

a {
  color: #00c3ff !important;
}

canvas {
  border: 5px solid #fff;
}

#prettydropdown-DefaultLang ul {
  color: #565d6e !important;
  background-color: #111316 !important;
}

.manual__step-one__header {
    color: #ffffff;
}
.manual__step-one__instructions {
    color: #ffffff;
}
```

**Comment by NicolasDorier** [comment]
And logo?

**Comment by pavlenex** [comment]
```
.top-header .header__icon {
    display: inline-block;
    vertical-align: middle;
    position: relative;
    width: 100px;
    height: 50px;
    margin-left: 10px;
    margin-top: 10px;
}
```
Not sure if you're referring to the source img, in that case

```
<img src="https://blockstream.com/img/icons/logo_blks.png" class="header__icon__img" height="40">
```

**Comment by NicolasDorier** [comment]
Should be good, anything else to change?

File: Theme.md [hunk 2]
```diff
@@ -30,3 +30,177 @@ Now, go into the settings of your store and paste the link to customize the boot
...
 
 ![CustomTheme](img/CustomTheme.png)
+
+## Checkout page theme
+
+It is possible to entirely customize the checkout experience of your customer with your brand.
+
+Go to your Store Settings, Checkout Experience, Link to a custom CSS stylesheet. Create your custom theme using these examples below:
+
+Maybe we can add something like:
```
Comment (britttttk) [inline Theme.md]: Remove :)

Comment (NicolasDorier) [reply Theme.md]: oops

**Comment by NicolasDorier** [comment]
Should be good now!

**Pull request merged**